stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/callback/StreamingCallableUnitCallBack.java

```diff
@@ -40,20 +41,21 @@ public StreamingCallableUnitCallBack(StreamObserver responseSender, ObserverCont
         this.responseSender = responseSender;
         this.observerContext = context;
     }
-    
+
     @Override
     public void notifySuccess() {
         super.notifySuccess();
     }
-    
+
     @Override
     public void notifyFailure(ErrorValue error) {
         if (responseSender != null) {
             handleFailure(responseSender, error);
         }
         if (observerContext != null) {
-            observerContext.addTag(TAG_KEY_HTTP_STATUS_CODE,
-                    String.valueOf(getMappingHttpStatusCode(Status.Code.INTERNAL.value())));
+            observerContext.addTag(TAG_KEY_HTTP_STATUS_CODE_GROUP,
+                    String.valueOf(getMappingHttpStatusCode(Status.Code.INTERNAL.value())).charAt(0) +
+                            STATUS_CODE_GROUP_SUFFIX);
```

Comment:
Same as in the other place we can use integer division. It might be even better to just hardcode "5xx"

Please do so in all similar places

Reply:
Fixed with f22240bfddcb1eaaa1c7c232136fd264c35b9b78